masterlist.yaml

```
@@ -3297,6 +3297,20 @@ plugins:
         itm: 35
         udr: 12
 
+  - name: 'Nec-Character_Clair.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/15581/' ]
```

Actually, I just realized that's a different mod.
https://www.nexusmods.com/oblivion/mods/15581/
is A CM Partner Companion Clair for Exnem Body.

https://www.nexusmods.com/oblivion/mods/15569/
is Nec Character_Clair

```suggestion
    url: [ 'https://www.nexusmods.com/oblivion/mods/15569/' ]
```

---

But on collection page this is the link listed:  http://www.tesnexus.com/downloads/file.php?id=15581

---

> But on collection page this is the link listed: http://www.tesnexus.com/downloads/file.php?id=15581 Or rather it should be http://www.nexusmods.com/oblivion/mods/15581

Right, it was probably added to the wrong file, `DAClair.esp` is that mod.

---

Nec01's Nec Character_Clair however is a base mod from which Divine Avenger's variant was made and later merged into one collection. Add both of them to the list perhaps?

---

> 
> 
> ~Nec01's Nec Character_Clair however is a base mod from which Divine Avenger's variant was made and later merged into one collection. Add both of them to the list perhaps?~ Well... yeah... need to looks at filenames

It's already there, the metadata for `Nec-Character_Clair.esp` was never correct.
`Nec-Character_Clair.esp` doesn't require CM Partners, so saying to use Sclerocephalus CM Partner Compilation instead doesn't make sense.